mixin.js

```diff
@@ -202,6 +202,19 @@
         return this;
       };
 
+      // Swipes
+      test.testCase.scroll = function functional_mixin_scroll(target, callback)
+      {
+        target = (typeof target == 'string' ? this.$(target) : target).first();
+        var position = this.$(target).offset().top + (this.$(target).outerHeight(true) / 2);
+
+        this.$('html, body').animate({
```

Comment:
And it would be nice to start it next tick, so in your test you can attach listeners before unleashing the mayhem.